packages/@holaplexui-playground/pages/form.tsx

```diff
@@ -1,67 +1,67 @@
   return (
     <svg
       width={size}
       height={size}
-      viewBox="0 0 16 16"
+      viewBox='0 0 16 16'
       fill={color}
       className={className}
-      xmlns="http://www.w3.org/2000/svg"
+      xmlns='http://www.w3.org/2000/svg'
     >
       <path
-        d="M9.41452 6.5865C10.1952 7.36717 10.1952 8.6345 9.41452 9.4165C8.63386 10.1972 7.36652 10.1972 6.58452 9.4165C5.80386 8.63583 5.80386 7.3685 6.58452 6.5865C7.36652 5.8045 8.63319 5.8045 9.41452 6.5865"
-        stroke="black"
-        stroke-linecap="round"
-        stroke-linejoin="round"
+        d='M9.41452 6.5865C10.1952 7.36717 10.1952 8.6345 9.41452 9.4165C8.63386 10.1972 7.36652 10.1972 6.58452 9.4165C5.80386 8.63583 5.80386 7.3685 6.58452 6.5865C7.36652 5.8045 8.63319 5.8045 9.41452 6.5865'
+        stroke='black'
+        stroke-linecap='round'
+        stroke-linejoin='round'
       />
       <path
-        fill-rule="evenodd"
-        clip-rule="evenodd"
-        d="M2 7.99992C2 7.56059 2.10133 7.12592 2.29733 6.72525V6.72525C3.30733 4.66059 5.53933 3.33325 8 3.33325C10.4607 3.33325 12.6927 4.66059 13.7027 6.72525V6.72525C13.8987 7.12592 14 7.56059 14 7.99992C14 8.43925 13.8987 8.87392 13.7027 9.27458V9.27458C12.6927 11.3393 10.4607 12.6666 8 12.6666C5.53933 12.6666 3.30733 11.3393 2.29733 9.27458V9.27458C2.10133 8.87392 2 8.43925 2 7.99992Z"
-        stroke="black"
-        stroke-linecap="round"
-        stroke-linejoin="round"
+        fill-rule='evenodd'
+        clip-rule='evenodd'
+        d='M2 7.99992C2 7.56059 2.10133 7.12592 2.29733 6.72525V6.72525C3.30733 4.66059 5.53933 3.33325 8 3.33325C10.4607 3.33325 12.6927 4.66059 13.7027 6.72525V6.72525C13.8987 7.12592 14 7.56059 14 7.99992C14 8.43925 13.8987 8.87392 13.7027 9.27458V9.27458C12.6927 11.3393 10.4607 12.6666 8 12.6666C5.53933 12.6666 3.30733 11.3393 2.29733 9.27458V9.27458C2.10133 8.87392 2 8.43925 2 7.99992Z'
+        stroke='black'
+        stroke-linecap='round'
+        stroke-linejoin='round'
       />
     </svg>
   );
 }
 
-function ShowPasswordIcon({ size = 16, color = "none", className = "" }) {
+function ShowPasswordIcon({ size = 16, color = 'none', className = '' }) {
   return (
     <svg
       width={size}
       height={size}
-      viewBox="0 0 16 16"
+      viewBox='0 0 16 16'
       fill={color}
       className={className}
-      xmlns="http://www.w3.org/2000/svg"
+      xmlns='http://www.w3.org/2000/svg'
     >
       <path
-        d="M9.70514 9.03858C9.27877 9.75749 8.45061 10.1358 7.62805 9.98746C6.80549 9.83911 6.16168 9.1953 6.01333 8.37274C5.86497 7.55018 6.2433 6.72202 6.96221 6.29565"
-        stroke="black"
-        stroke-linecap="round"
-        stroke-linejoin="round"
+        d='M9.70514 9.03858C9.27877 9.75749 8.45061 10.1358 7.62805 9.98746C6.80549 9.83911 6.16168 9.1953 6.01333 8.37274C5.86497 7.55018 6.2433 6.72202 6.96221 6.29565'
+        stroke='black'
+        stroke-linecap='round'
+        stroke-linejoin='round'
```

Comment:
This dashes aren't valid prop types in react can you find and replace them all to `storkeLinecap` and `strokeLinejon` this is something we need to do with every svg all the dash casing needs to be converted to camel.